appnav/src/main/kotlin/io/element/android/appnav/LoggedInFlowNode.kt

```diff
@@ -375,6 +381,20 @@ class LoggedInFlowNode @AssistedInject constructor(
                     })
                     .build()
             }
+            is NavTarget.IncomingShare -> {
+                shareEntryPoint.nodeBuilder(this, buildContext)
+                    .callback(object : ShareEntryPoint.Callback {
+                        override fun onDone(roomIds: List<RoomId>) {
+                            navigateUp()
+                            if (roomIds.size == 1) {
```

Comment:
At the moment this is always the case, right? Do we want to limit sharing to just 1 room at a time?